ex03.exs

```diff
@@ -62,9 +62,19 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    n = collection |> Enum.count |> div(process_count)
+
```

Comment:
I like this. 

(You don't need the module name on calls to functions in the same module)